eventstore/jpa/spring-data/src/main/java/org/occurrent/eventstore/jpa/springdata/StreamEntity.java

```diff
@@ -0,0 +1,49 @@
+package org.occurrent.eventstore.jpa.springdata;
+
+import jakarta.persistence.*;
+
+import java.util.UUID;
+
+
+@Entity
+@Table(name = "event_stream",
+        indexes = {
+                @Index(name = "idx_event_stream_name", columnList = "name", unique = true)
+        })
+public class StreamEntity {
```

Comment:
I guess this class is internal and should not be used outside by users of the Occurrent lib? If so, I think that either it should be moved to an "internal" package or remove "public" from the interface to make this more clear.